vm/tests/solutions/spec/jenkins/ports_spec.rb

```diff
@@ -19,11 +19,11 @@
 end
 
 describe port(80) do
-  it { should be_listening.on('::').with('tcp6') }
```

Comment:
Nit, this change shouldn't be needed for debian10 ;)

Reply:
Debian 10 returns *:80 for IPv4 and IPv6, but specinfra can't validate this format. 

References:
* https://git.kernel.org/pub/scm/network/iproute2/iproute2.git/commit/?id=1267c0b924de0297510efd48f8f849af774be369
* https://github.com/mizzy/specinfra/blob/master/lib/specinfra/command/module/ss.rb#L31